process/sync/storageBootstrap/baseStorageBootstrapper.go

```diff
@@ -296,6 +298,12 @@ func (st *storageBootstrapper) applyHeaderInfo(hdrInfo bootstrapStorage.Bootstra
 		return err
 	}
 
+	err = st.getAndApplyProofForHeader(headerHash, headerFromStorage)
```

Comment:
can you move this call inside the st.applyBlock instead, and remove the forkdetector checkpoint update from getAndApplyProofForHeader and leave it at the end in applyBlock instead?

Reply:
updated as it was before the changes